scripts/generate_prefrosh.py

```diff
@@ -0,0 +1,18 @@
+import sqlalchemy
+import os
+from ruddock import config
+
+engine = sqlalchemy.create_engine(config.DEV.db_uri, convert_unicode=True)
+
+if __name__ == "__main__":
+  for filename in os.listdir('../../images/'):
+      full_name = os.path.splitext(filename)
+      last, dash, first = full_name[0].partition('-')
```

Comment:
split() ?

Also in the documentation (see above) definitely make sure to note how the filenames are being parsed and that manual editing may be necessary for names that have hyphens in them.